host/sgx/sgxquoteprovider.h

```diff
@@ -25,9 +25,7 @@ typedef struct _oe_sgx_quote_provider
         free_sgx_quote_verification_collateral;
 } oe_sgx_quote_provider_t;
 
-bool oe_set_quote_provider_logger(
-    oe_sgx_quote_provider_t* provider,
-    sgx_ql_logging_function_t logger);
+bool oe_sgx_set_quote_provider_logger(sgx_ql_logging_function_t logger);
```

Comment:
Per https://github.com/openenclave/openenclave/blob/master/docs/ApiGuidelines.md is this API
public, experimental, or internal?  If internal only, this change is ok, but it would be good to add a comment at the top of the file saying that this file is internal and should not be used by applications since APIs in this file are not stable.

Reply:
Sure, let me add some comment to it, this should not be used externally